Minor documentation updates

ajazam opened the pull request:
I corrected some minor issues in the documentation that were caused by me. If there are any problems with these changes please let me know.

lingmann commented on 1.7/administration/installing/cloud/gce.md:
````diff
@@ -22,10 +22,10 @@ A bootstrap node is required to run the scripts and to bootstrap the DC/OS clust
 
     ```bash
-    $ gcloud components update &&
+    $ sudo gcloud components update &&
````
I believe that best practices are to have gcloud installed with user privs, so running the components update as root should not be necessary nor desirable, as that would mutate the privileges of the user installed gcloud instance.

ajazam replied:
@lingmann, you are right about that. It should say

gcloud components update &&


I have also tested this on a micro instance and it does work with the sudo.
I'm relatively new to using github with a community so if there is anything
I need todo or for some reason I don't respond then please use my name and
I will respond :)

On Wed, 3 Aug 2016 at 23:27 lingmann <notifications@github.com> wrote:

> In 1.7/administration/installing/cloud/gce.md
> <https://github.com/dcos/dcos-docs/pull/476#discussion_r73433163>:
>
> >
> >      ```bash
> > -    $ gcloud components update &&
> > +    $ sudo gcloud components update &&
>
> I believe that best practices are to have gcloud installed with user
> privs, so running the components update as root should not be necessary nor
> desirable, as that would mutate the privileges of the user installed gcloud
> instance.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/dcos/dcos-docs/pull/476/files/d64eae18df9c173bdebb857a47e92312f41d7068#r73433163>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABwTc8OQUEIh7tjxYynpQFB7tHsPeKlEks5qcRW2gaJpZM4JY_a8>
> .
>
-- 
Regards

Abdul Jabbar Azam
twitter: @ajazam

lingmann commented on 1.7/administration/installing/cloud/gce.md:
````diff
@@ -54,7 +54,7 @@ You must create the RSA public/private keypairs to allow passwordless logins via
 
     ```bash
-    $ vi ~/.ssh/id_rsa.pub
+    $ sudo vi ~/.ssh/id_rsa.pub
````
`~/.ssh/id_rsa.pub` should be owned by the current user, so sudo not necessary here.

lingmann commented on 1.7/administration/installing/cloud/gce.md:
````diff
@@ -145,7 +145,7 @@ You must create the RSA public/private keypairs to allow passwordless logins via
     ```
 
-1.  Review and customize the `dcos/gce/group_vars/all`. You should review `project`, `subnet`, `login_name`, `bootstrap_public_ip`, and `zone`.
+1.  Review and customize the `dcos_gce/group_vars/all`. You should review `project`, `subnet`, `login_name`, `bootstrap_public_ip`, and `zone`.
````
@pyronicide can you review?

pyronicide commented:
@ajazam have you had a chance to look at @lingmann's comments?

ajazam commented:
Sorry I didn't realise the comments were for me. I'll look at them when I
get home.

On Mon, 8 Aug 2016, 15:21 Thomas Rampelberg, <notifications@github.com>
wrote:

> @ajazam <https://github.com/ajazam> have you had a chance to look at
> @lingmann <https://github.com/lingmann>'s comments?
>
> —
> You are receiving this because you were mentioned.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/dcos/dcos-docs/pull/476#issuecomment-238252785>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/ABwTc_Xl430UeRmOjM1R3VaZGGWGYyK7ks5qdztwgaJpZM4JY_a8>
> .
>
-- 
Regards

Abdul Jabbar Azam
twitter: @ajazam

ajazam commented:
Is there a problem with the pull request? I'll be making more pull requests with changes. Can someone provide some feedback please?

mhausenblas commented:
@ajazam thanks a lot for this! Any chance that you can also address [DCOS-434](https://dcosjira.atlassian.net/browse/DCOS-434) with this PR?

ajazam commented:
I have been looking at this. I'm having a few problems with docker. I will have a look again tonight.

ajazam commented:
It appears to be working now. I've changed the docker version and the docker.service file. I'll do some more testing tonight and push the latest changes. I'll create a PR for the minor documentation changes for this.

ajazam commented:
Some of the services were having problems starting so I have increased the core count to two per VM and now all the services are starting.  DCOS-434 has now been addressed. The documentation in the github repository is correct.

However I'm not sure how to apply the changes to the documentation on the main dcos site. Do I create a new PR?

mhausenblas commented:
Thanks a lot for addressing DCOS-434 @ajazam and I think we're good for final review/editing and publishing @joel-hamill? (Note: I believe the 1.8 docs require an update as well?)

ajazam commented:
@mhausenblas I need to modify the docs and explain that the install scripts will use the latest stable version of dcos, 1.8.2 . I have also made other changes e.g. the VM's used by the script will use n2-standard-1. I've only tested the changes in 1.8.2. Do you want me to test the changes with 1.7 and master?

I think I need to do a PR to apply the changes to 1.7, 1.8 and 1.9 docs.

joel-hamill commented:
@ajazam 
> re: I think I need to do a PR to apply the changes to 1.7, 1.8 and 1.9 docs. 
You should be able to commit those changes to this same PR, if you prefer.

ajazam commented:
Sorry for the delay. I was modifying the docs and the script on my Mac and also making changes to the script on the bootstrap instance and not syncing the changes properly. This caused soakled up lots of time.

I think I've added the new alterations. The changes work with 1.7-open and 1.8.4 I'm not sure if 1.9 is available. I've added an appropriate comment. If somebody can supply the link to the installer I can 1.9

I think somebody in the past commented on my use of 

>$ sudo gcloud components update

If I remove the sudo statement then the bootstrap node returns the following error

> ERROR: (gcloud.components.update) You cannot perform this action because you do not have permission to modify the Google Cloud SDK installation directory [/usr/local/share/google/google-cloud-sdk].
> 
> Re-run the command with sudo: sudo /usr/local/share/google/google-cloud-sdk/bin/gcloud ...

ajazam commented:
The changes also work with v1.9-dev.

joel-hamill commented:
LGTM

ajazam commented:
Hello. Can somebody please merge this pull request? It would be good to see the website showing the correct instructions :)

mhausenblas commented:
Thanks, again @ajazam!
